consul_exporter.go

```diff
@@ -61,6 +61,11 @@ var (
 		"How many members are in the cluster.",
 		nil, nil,
 	)
+	memberStatus = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, "", "members_status"),
+		"Member status in the cluster.",
+		[]string{"dc", "node"}, nil,
```

Comment:
The other metrics don't include any `dc` label, lets stick with this rule. IIUC it can (should?) be added during service discovery.

Reply:
OK, I will remove dc label.